Classes/CZ805MFire.uc

```diff
@@ -5,22 +5,23 @@ defaultproperties
      FireSoundRef="ScrnWeaponPack_SND.cz805.Firecz805b"
      NoAmmoSoundRef="KF_SCARSnd.SCAR_DryFire"
 
-     FireAimedAnim="Fire"
+     FireAimedAnim="Fire_Iron"
      RecoilRate=0.060000
-     maxVerticalRecoilAngle=125
+     maxVerticalRecoilAngle=150
      maxHorizontalRecoilAngle=75
-     ShellEjectClass=Class'ROEffects.KFShellEjectSCAR'
+     //ShellEjectClass=Class'ROEffects.KFShellEjectSCAR'
+     ShellEjectClass=class'ScrnWeaponPack.CZ805MShellEject' //changed to m4
      ShellEjectBoneName="Shell_eject"
      bAccuracyBonusForSemiAuto=True
      DamageType=Class'ScrnWeaponPack.DamTypeCZ805M'
-     DamageMax=35
+     DamageMax=41
```

Comment:
Why did you buff CZ805 damage? The initial idea was to have its low base damage and high headshot multiplier. With all your visual fixes to the weapon, I think it is even more important to stay like that. 
If you want to make it stronger, consider raising headshot multiplier instead of base damage.

Reply:
Your arguments sound reasonable. OK, let's apply your changes.